packages/orchestration/src/types.ts

```diff
@@ -11,3 +11,20 @@ export type * from './orchestration-api.js';
 export type * from './exos/cosmos-interchain-service.js';
 export type * from './exos/chain-hub.js';
 export type * from './vat-orchestration.js';
+
+/**
+ * ({@link ZCF})-like tools for use in {@link OrchestrationFlow}s.
+ *
+ * @interface
+ */
+export type ZcfTools<CT = Record<string, unknown>> = Pick<
+  ZCF<CT>,
```

Comment:
I don't think we care about `CT` since from what I gather that only shows up in terms which are not picked

```suggestion
export type ZcfTools = Pick<
  ZCF,
```